### PR TITLE
Docs: update the Overflow docs with information about forwarding ref when custom component is rendered

### PR DESCRIPTION
**Pull request by george-cz**
Fixes [#27652](https://github.com/microsoft/fluentui/issues/27652)

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports | Baseline (minified/GZIP) | PR    | Change     |
| :---------------- | -----------------------: | ----: | ---------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `221.733 kB`<br />`62.557 kB` | `221.825 kB`<br />`62.568 kB` | `92 B` <br />`11 B` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ReactComponents.fixture.js'>react-components: entire library</abbr> | `1.101 MB`<br />`267.232 kB` | `1.101 MB`<br />`267.249 kB` | `92 B` <br />`17 B` |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports | Size (minified/GZIP) |
| ----------------- | -------------------: |
| <samp>react-breadcrumb</samp> <br /> <abbr title='bundle-size/index.fixture.js'>@fluentui/react-breadcrumb - package</abbr> | `117.263 kB`<br />`32.241 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr> | `71.55 kB`<br />`20.584 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr> | `44.037 kB`<br />`14.418 kB` |
| <samp>react-overflow</samp> <br /> <abbr title='bundle-size/hooks.fixture.js'>hooks only</abbr> | `12.86 kB`<br />`4.825 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr> | `8.39 kB`<br />`2.64 kB` |
</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/92ea818bca050e7d2ee889b9a1716de110d05f0d'>92ea818bca050e7d2ee889b9a1716de110d05f0d</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31292/perf-test-react-components/Avatar-mount_base.html">663</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31292/perf-test-react-components/Avatar-mount.html">694</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31292/perf-test-react-components/Button-mount_base.html">318</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31292/perf-test-react-components/Button-mount.html">302</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31292/perf-test-react-components/Field-mount_base.html">1128</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31292/perf-test-react-components/Field-mount.html">1121</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31292/perf-test-react-components/FluentProvider-mount_base.html">693</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31292/perf-test-react-components/FluentProvider-mount.html">707</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31292/perf-test-react-components/FluentProviderWithTheme-mount_base.html">78</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31292/perf-test-react-components/FluentProviderWithTheme-mount.html">88</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31292/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">30</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31292/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">36</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31292/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">83</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31292/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">70</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31292/perf-test-react-components/MakeStyles-mount_base.html">864</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31292/perf-test-react-components/MakeStyles-mount.html">866</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31292/perf-test-react-components/Persona-mount_base.html">1770</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31292/perf-test-react-components/Persona-mount.html">1691</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31292/perf-test-react-components/SpinButton-mount_base.html">1407</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31292/perf-test-react-components/SpinButton-mount.html">1409</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31292/perf-test-react-components/SwatchPicker-mount_base.html">1529</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/31292/perf-test-react-components/SwatchPicker-mount.html">1564</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: packages/react-components/react-overflow/stories/Overflow/CustomComponent.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import {
```
Comment (fabricteam) [inline packages/react-components/react-overflow/stories/Overflow/CustomComponent.stories.tsx:1]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31292&commitId=717952ed69dbdcdca9d323580743ebce90ca7dda&env=prod&clientType=FLUENTUI)


  <details><summary><b>Positioning</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning.auto size overflow padding - RTL.chromium.png|0| Added|
|Positioning.auto size overflow padding shorthand.chromium.png|0| Added|
|Positioning.auto size overflow padding.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Review by layershifter — APPROVED** [review]

**Pull request merged**